### ISSUE TITLE
Navy ignores bounty hunters

### ISSUE THREAD

**Issue by paulusasol** [endless-sky/endless-sky#5812]
#### Observed behavior:
Navy ships ignore bounty hunters attacking the player, e.g. during mystery cube missions.

#### Expected behavior:
Navy ships should not ignore the attempted (or successful) murder of the player unless there is a specific reason not to interfere. This should also be true for similar organizations, e.g. Deep Security or Free Worlds militia ships.

#### Justification
While the bounty hunters just want to kill the player instead of stealing cargo, why would that be a reason for the navy to just watch?

There may of course exceptions for this, e.g. if the player a pirate and thus the navy does not care.

This gives the player a third option to dealing with too large bounty hunter fleets:
Instead of being faster than the bounty hunters or cancelling the mission causing them, they can just fly into a system with significant navy activity.

**Comment by Zitchas** [comment]
I agree with this - Bounty hunters occupy a weird space in terms of criminality. Unlike pirates, the navy (and other "police" forces) shouldn't automatically attack them. But when they engage in actions (such as attacking a merchant) that *are* against their governmental settings, they *should* be attacked. 

Ideally, this would take into account the relevant government settings, for instance, hostility and/or alliances.

Ex. The Republic would step in to help a member of a government that they have a positive relationship with (merchants, militia, navy intelligence, navy oathkeeper, and neutrals), but they would ignore attacks on members of government that they don't have a value for.

That being said, this might tie into a larger revamp of the government system, as there are a number of flaws and inconsistencies with it.

**Comment by Zavindur** [comment]
From the lore side, legal bounty hunters are those who go after people who posted bail but did not show up for their court dates; it'd make sense if local security was like "we're not getting involved in that" because while it's technically attacking another ship- there might be a legal reason to do so.

**Comment by Zitchas** [comment]
That's true, on the other hand, I am under the impression that if legal bounty hunters start shooting up someone's car, the police are going to get involved regardless. In most jurisdictions, police generally have a "get involved first, ask questions later" policy in regards to violence. Because within their jurisdiction *no-one* should be shooting up anything without involving the police first. Even the military in many countries can't just pull out their guns and blast someone without clear national threat justification.

**Comment by Zavindur** [comment]
So it seems the lore would need to bridge that gap somehow;  I have heard that in some cultures and places, the police are more like mall cops and run from the gun shots instead of running to them- so not everyone tries to be a hero; but it would seem odd to have a seemingly smaller/independent force (bounty hunters) to have the established security force step aside.

Maybe in the future, a "License to Kill" becomes a real thing?

**Comment by Zitchas** [comment]
I think the predominant lore is probably that - legally speaking - laws & national authority are considered to end at the edge of the atmosphere. So Space counts as a grey zone that might be considered "within a faction's sphere of influence" but isn't actually part of the country. Thus all of space becomes something akin to international waters. While we have a variety of "laws" governing what is and is not legal in international waters, in practical terms it comes down to the captain of a ship being the rule of law, whatever they deem it to be. Along with "might makes right." AKA, whoever has the biggest gun and the will to use it rules.

So we basically have three "spheres":
- Planets: Controlled and owned by governments, the actual national territory. (full laws apply here, including things like police having a monopoly on violence, murdering a criminal will get you charged with murder, etc.)
- Systems: Inhabited. Not actual national territory, but generally controlled by whoever considers it "their" territorial space. (limited application of civil/criminal law here, strong governance of national and security policy. Military holds dominant control of violence, but other parties may use it. Might boil down to "so long as you just use it against our enemies, we're fine with it." sort of attitudes.)
- Systems: Uninhabited/uncontrolled: No-man's land where no law applies whatsoever. Most individuals will be at least loosely bound by the laws of their respective nationalities, but it is weak. (no concrete legal systems. Individuals from similar legal backgrounds may choose to continue to treat each other as being under those laws, but enforcement is up to them. Example, two people from the respublic will probably treat each other normally. If one robs the other, the victim might try to bring it to court in republic space))

Also worth noting that we don't actually see police at any point in time. We just deal with various Navy and military forces. Historically, militaries are *NOT* law enforcement. They are instruments of sovereignty and the application of overwhelming lethal force. They'd prefer you kept their law, but in international waters they're going to primarily be focused on the basics. "You're not killing us, our friends, or merchants? Then we don't care what you're doing."

So we basically have a sliding scale from Planets, with the strongest legal restrictions to uninhabited systems with zero legal systems.
